setup_monorepo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+# start building packages
+echo "Installing dependencies"
+cd ethereumjs-monorepo
+npm i
```

Comment:
```suggestion
pnpm i
```

Reply:
it is monorepo with npm, if we want to switch to pnpm, we need another file for declaring workspace, and doing so would make change the submodule as well